### PR TITLE
Update CI_Workflow.yml

### PR DESCRIPTION
**Pull request by RafaelOrti**
CI GITHUB workflow added

### CODE REVIEW

**Review by RafaelOrtiPython — COMMENTED** [review]
This should fix the node.js version problem, but fix the error in the working-directory field

**Review by RafaelOrti — COMMENTED** [review]
Error in the working-directory fixed in github

File: .github/workflows/CI_Workflow.yml [hunk 1]
```diff
@@ -11,7 +11,7 @@ jobs:
 
     strategy:
       matrix:
-        node-version: [14.x, 16.x, 18.x]
+        node-version: [21.x]
```
Comment (RafaelOrtiPython) [inline .github/workflows/CI_Workflow.yml:14]: This version should work fine

**Review by RafaelOrtiPython — COMMENTED** [review]
working-directory fixed correctly

**Review by RafaelOrtiPython — COMMENTED** [review]
Changes updated correctly

**Review by RafaelOrtiPython — APPROVED** [review]
Errors fixed correctly

**Pull request merged**